bool[] occupancyGrid
int16[] gridSize
float32 resolution
int16 ndims
